api-compose-object.go

```diff
@@ -127,7 +127,7 @@ type SourceInfo struct {
 // `decryptSSEC` is the decryption key using server-side-encryption
 // with customer provided key. It may be nil if the source is not
 // encrypted.
-func NewSourceInfo(bucket, object string, sse encrypt.ServerSide) SourceInfo {
+func NewSourceInfo(bucket, object string, sse encrypt.ServerSide, replaceTags bool, tags map[string]string) SourceInfo {
```

Comment:
Actually I think you should not make this change instead it should be changed in CopyObjectOptions{} where we set new metadata